src/main/frontend/fs.cljs

```diff
@@ -127,8 +127,13 @@
 
 (defn open-dir
   [ok-handler]
-  (let [record (get-record)]
-    (p/let [result (protocol/open-dir record ok-handler)]
+  (let [record (get-record)
+        ;; it is not easy to invoke `ls-dir-files` in electron
+        ;; we add a secret backdoor here for testing
+        global-node-dir (or (.-secretlogseqgraphdir js/window) nil)]
```

Comment:
since this is special enough, you might use a `SNAKE_CASE`, guard these logic behind a `MOCK` environment variable.

Reply:
Can you suggest how to set/consume a `MOCK` environment variable in the browser context?

Reply:
In building stage of :electron target, this might be possible.
Using `goog-define` and shadow-cljs.json

```clojure
electron.conf/MOCK #shadow/env ["MOCK" :as :bool :default false]
```

ref: https://shadow-cljs.github.io/docs/UsersGuide.html#shadow-env

There's a `DEV-RELEASE` config used in `src/main/frontend/config.cljs`.

Reply:
I refactored the implementation by reading the environment variable `LOGSEQ_OVERWRITE_OPEN_DIR`, also added a test for loading local graph dir & checking a saved page.
I think overwriting this value in compile time is too restricted for this case. What do you think?